synapseclient/__main__.py

```diff
@@ -545,28 +537,30 @@ def build_parser():
 
     parser_submit = subparsers.add_parser('submit',
             help='submit an entity or a file for evaluation')
-    parser_submit.add_argument('--evaluationID', '--evalID', type=str,
+    parser_submit.add_argument('--evaluationID', '--evaluationId', '--evalID', type=str,
             help='Evaluation ID where the entity/file will be submitted')
     parser_submit.add_argument('--evaluationName', '--evalN', type=str,
             help='Evaluation Name where the entity/file will be submitted')
     parser_submit.add_argument('--evaluation', type=str,
             help=argparse.SUPPRESS)  #mainly to maintain the backward compatibility
-    parser_submit.add_argument('--entity', '--eid', '--entityId', type=str,
+    parser_submit.add_argument('--entity', '--eid', '--entityId', '--id', type=str,
             help='Synapse ID of the entity to be submitted')
     parser_submit.add_argument('--file', '-f', type=str,
             help='File to be submitted to the challenge')
-    parser_submit.add_argument('--parentId', '--pid', type=str, dest='parentid',
+    parser_submit.add_argument('--parentId', '--parentid', '--parent', type=str, dest='parentid',
```

Comment:
I don't think parent is necessary to put explicitly as it should already work as as does --pare and --pa etc.